node/service/chain-specs/example.json

```diff
@@ -95,6 +95,36 @@
           ]
         ]
       },
+      "democracy": {
+        "phantom": null
+      },
+      "council": {
+        "phantom": null,
+        "members": []
+      },
+      "technicalCommittee": {
+        "phantom": null,
+        "members": [
+          "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+          "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty"
+        ]
+      },
+      "elections": {
```

Comment:
The `elections` pallet was removed when we introduced OpenGov, you should remove it from the example.json file.